consensus/consortium/v2/snapshot.go

```diff
@@ -263,9 +265,10 @@ func (s *Snapshot) apply(headers []*types.Header, chain consensus.ChainHeaderRea
 
 				oldLimit := len(snap.validators())/2 + 1
 				var newLimit int
-				// After Tripp, list of block producers is retrieved from the
```

Comment:
Overall, I think this change does not relates to this PR. This PR should be about query contract by using block hash, any other changes should be in a separate PR.